lib/rubocop/ast/processed_source.rb

```diff
@@ -99,22 +99,39 @@ def blank?
         ast.nil?
       end
 
+      # @return [Comment, nil] the comment at that line, if any.
+      def comment_at_line(line)
```

Comment:
I guess this covers both whole line comments and inline comments, right?

Reply:
Yes! API that applies only to whole line comments should be pretty rare imo